docs/source/getting_started/authoring_your_workflow.md

```diff
@@ -18,91 +18,85 @@ The tutorial aims to be an extension to the quickstart to help you better unders
+
 
 ```python
+from latch.types import LatchFile, LatchOutputDir
+...
+
 @small_task
-def assembly_task(read1: LatchFile, read2: LatchFile) -> LatchFile:
-
-    # A reference to our output.
-    sam_file = Path("covid_assembly.sam").resolve()
-
-    _bowtie2_cmd = [
-        "bowtie2/bowtie2",
-        "--local",
-        "-x",
-        "wuhan",
-        "-1",
-        read1.local_path,
-        "-2",
-        read2.local_path,
-        "--very-sensitive-local",
-        "-S",
-        str(sam_file),
-    ]
-
-    subprocess.run(_bowtie2_cmd)
-
-    return LatchFile(str(sam_file), "latch:///covid_assembly.sam")
-```
+def assembly_task(
+    read1: LatchFile, read2: LatchFile, output_directory: LatchOutputDir
+) -> LatchFile:
 
-### How to work with LatchFiles
+    local_file = Path(read1)
+    ...
+```
 
-`LatchFile` refers to a remote file location on
-[Latch Console](https://console.latch.bio/data).
+Sometimes, a python task does not directly access a file/directory, but uses it implicitly; for example, through a subprocessed shell command. We can ensure that the file's data is present by either calling `Path` on the file or directory, or by accessing the `local_path` attribute of the LatchFile or LatchDir. This will download the data into the task environment and return back a path to the local file or directory.
 
-To make the file available to the local task's environment, you have to call `local_path` property like so:
 ```python
-_bowtie2_cmd = [
-        "bowtie2/bowtie2",
-        "--local",
-        "-x",
-        "wuhan",
-        "-1",
-        read1.local_path,
-        "-2",
-        read2.local_path,
-        "--very-sensitive-local",
-        "-S",
-        str(sam_file),
-    ]
+bowtie2_cmd = [
+    "bowtie2/bowtie2",
+    "--local",
+    "--very-sensitive-local",
+    "-x",
+    "wuhan",
+    "-1",
+    read1.local_path,
+    "-2",
+    read2.local_path,
+    "-S",
+    str(sam_file),
+]
 ```
 
-Once the subprocess finishes executing the command above, it creates an output file (`str(sam_file)`).
-
-To make the SAM file available to Latch Console, you have to upload it using the following return statement:
+A file or directory can be outputted to the Latch platform as below:
 ```python
-    return LatchFile(str(sam_file), "latch:///covid_assembly.sam")
+    local_file = Path("/root/covid_assembly.sam")
+    return LatchFile(local_file, "latch:///covid_assembly.sam")
 ```
 
-Here, LatchFile takes two values: the first being your local filepath and the second being the remote file location on Latch. Every file on Latch must be prefixed with `latch:///`.
+Here, `LatchFile()` takes as input two values: the first being the local filepath and the second being the desired remote file location on Latch. Every file on Latch must have the prefix `latch:///`.
```

Comment:
suggestion: `LatchFile(...)`

"the first being" -> "the first is"
"the second being" -> "the second is"
"desired" -> "target"

the prefix thing is strictly speaking wrong because we have other prefixes (in new lytekit) but doesn't need to be changed yet
"every file on Latch must have" -> "Remote paths on Latch start with"